src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -2032,47 +2602,57 @@ void EnGo_TransfromLimbDraw(PlayState* play, s32 limbIndex, Actor* thisx) {
     }
 
     switch (limbIndex) {
-        case 17:
-            SubS_UpdateLimb(this->unk_3B0 + this->unk_3B4 + 0x4000,
-                            this->unk_3B2 + this->unk_3B6 + this->actor.shape.rot.y + 0x4000, &this->unk_290,
-                            &this->unk_2A8, stepRot, overrideRot);
+        case GORON_LIMB_HEAD:
+            SubS_UpdateLimb(this->headRotZ + this->bodyRotZ + 0x4000,
+                            this->headRotY + this->bodyRotY + this->actor.shape.rot.y + 0x4000, &this->headPos,
+                            &this->headRot, stepRot, overrideRot);
             Matrix_Pop();
-            Matrix_Translate(this->unk_290.x, this->unk_290.y, this->unk_290.z, MTXMODE_NEW);
+            Matrix_Translate(this->headPos.x, this->headPos.y, this->headPos.z, MTXMODE_NEW);
             Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
-            Matrix_RotateYS(this->unk_2A8.y, MTXMODE_APPLY);
-            Matrix_RotateXS(this->unk_2A8.x, MTXMODE_APPLY);
-            Matrix_RotateZS(this->unk_2A8.z, MTXMODE_APPLY);
+            Matrix_RotateYS(this->headRot.y, MTXMODE_APPLY);
+            Matrix_RotateXS(this->headRot.x, MTXMODE_APPLY);
+            Matrix_RotateZS(this->headRot.z, MTXMODE_APPLY);
             Matrix_Push();
             break;
 
-        case 10:
-            SubS_UpdateLimb(this->unk_3B4 + 0x4000, this->unk_3B6 + this->actor.shape.rot.y + 0x4000, &this->unk_29C,
-                            &this->unk_2AE, stepRot, overrideRot);
+        case GORON_LIMB_BODY:
+            SubS_UpdateLimb(this->bodyRotZ + 0x4000, this->bodyRotY + this->actor.shape.rot.y + 0x4000, &this->bodyPos,
+                            &this->bodyRot, stepRot, overrideRot);
             Matrix_Pop();
-            Matrix_Translate(this->unk_29C.x, this->unk_29C.y, this->unk_29C.z, MTXMODE_NEW);
+            Matrix_Translate(this->bodyPos.x, this->bodyPos.y, this->bodyPos.z, MTXMODE_NEW);
             Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
-            Matrix_RotateYS(this->unk_2AE.y, MTXMODE_APPLY);
-            Matrix_RotateXS(this->unk_2AE.x, MTXMODE_APPLY);
-            Matrix_RotateZS(this->unk_2AE.z, MTXMODE_APPLY);
+            Matrix_RotateYS(this->bodyRot.y, MTXMODE_APPLY);
+            Matrix_RotateXS(this->bodyRot.x, MTXMODE_APPLY);
+            Matrix_RotateZS(this->bodyRot.z, MTXMODE_APPLY);
             Matrix_Push();
             break;
     }
 }
 
-void func_80A15FEC(Actor* thisx, PlayState* play) {
+/**
+ * Draw function
+ *
+ * Draws the goron in any/all forms, and the effects swawned off of them.
+ * Examples include:
+ * - Regular animations
+ * - Gorons Encased in ice (Gatekeeper and GraveDigger's brother)
+ * - Gatekeeper's rolling snowball
+ * - Snow, Dust, and Steam effects coming off gorons (When breaking the snowball or melting the ice)
+ */
```

Comment:
You can probably just say "draws everything relating to this actor", or "Draws the Goron and associated effects etc." Don't need to mention the animations, that's completely standard.